tests/e2e/runner/runner.go

```diff
@@ -92,8 +92,9 @@ func startRunner(vmName string, genesisPath string, pluginDir string) error {
 	return nil
 }
 
-func WaitForCustomVm(vmId ids.ID) (string, string, error) {
+func WaitForCustomVm(vmId ids.ID) (string, string, int, error) {
```

Comment:
Can we move this logic inside of the avalanche-network-runner? I would think that we can move the for loop here into the avalanche-network-runner and simplify this code which will likely need to be duplicated in any project doing this similar and most likely very common task.

Reply:
This is already included for second ANR refactoring, but will anyway make a explicit issue now and work first on it. https://github.com/ava-labs/avalanche-network-runner/issues/302
After doing that, will add issues to modify CLI/subnet-evm